### ISSUE TITLE
Non-default parameter values immediately after migration

### ISSUE THREAD

**Issue by Shekelme** [filecoin-project/boost#1594]
### Checklist

- [X] This is **not** a question or a support request. If you have any boost related questions, please ask in the [discussion forum](https://github.com/filecoin-project/boost/discussions).
- [X] This is **not** a new feature request. If it is, please file a [feature request](https://github.com/filecoin-project/boost/issues/new?assignees=&labels=need%2Ftriage%2Ckind%2Ffeature&template=feature_request.yml) instead.
- [X] This is **not** an enhancement request. If it is, please file a [improvement suggestion](https://github.com/filecoin-project/boost/issues/new?assignees=&labels=need%2Ftriage%2Ckind%2Fenhancement&template=enhancement.yml) instead.
- [X] I **have** searched on the [issue tracker](https://github.com/filecoin-project/boost/issues) and the [discussion forum](https://github.com/filecoin-project/boost/discussions), and there is no existing related issue or discussion.
- [X] I am running the [`Latest release`](https://github.com/filecoin-project/boost/releases), or the most recent RC(release canadiate) for the upcoming release or the dev branch(master), or have an issue updating to any of these.
- [X] I did not make any code changes to boost.

### Boost component

- [X] boost daemon - storage providers
- [ ] boost client
- [ ] boost UI
- [ ] boost data-transfer
- [ ] boost index-provider
- [ ] Other

### Boost Version

```text
boostd version 1.7.4+git.52cb7cd
```

### Describe the Bug

I observe non-default parameter values in `config.toml` immediately after migration. At least one of these parameters (`StorageListRefreshDuration = "0s"`) prevents the boost from starting. The second parameter - `RedeclareOnStorageListRefresh = false`.
Perhaps there are others.

### Logging Information

```text
2023-04-24T18:07:55.557+0900    DEBUG   boostd  boostd/run.go:95        Instantiating new boost node
2023-04-24T18:07:55.571+0900    INFO    modules modules/rcmgr.go:44     libp2p resource manager is enabled
2023-04-24T18:07:55.571+0900    INFO    modules modules/rcmgr.go:92     adjusted default resource manager limits
2023-04-24T18:07:55.571+0900    DEBUG   rcmgr   resource-manager/limit.go:84    initializing new limiter with config    {"limits": {}}
2023-04-24T18:07:55.600+0900    INFO    modules modules/storageminer_svc.go:67  Connecting storage service to miner
2023-04-24T18:07:55.601+0900    INFO    modules modules/storageminer_svc.go:30  Checking (svc) api version of ws://xxx.xxx.x.xxx:2345/rpc/v0
2023-04-24T18:07:55.605+0900    INFO    modules modules/storageminer_svc.go:60  Connecting sealing service to miner
2023-04-24T18:07:55.605+0900    INFO    modules modules/storageminer_svc.go:30  Checking (svc) api version of ws://xxx.xxx.x.xxx:2345/rpc/v0
2023-04-24T18:08:01.609+0900    INFO    modules modules/storageminer_idxprov.go:114     Instantiated index provider engine      {"idxProvEnabled": true, "pid": "12D3KooWDaPPCdpbzt3fwKG93qFyAokKwiHLSE56nrjd7ZQ3AAna", "topic": "/indexer/ingest/mainnet", "retAddrs": ["/ip4/58.72.32.242/tcp/9191"], "extraGossipData": "f01137229", "publisher": "data-transfer"}
2023-04-24T18:08:01.609+0900    INFO    modules modules/storageminer.go:377     starting legacy storage provider
2023-04-24T18:08:01.612+0900    INFO    modules modules/storageminer.go:440     Contract deals monitor is currently disabled. Update config.toml if you want to enable it.
2023-04-24T18:08:01.615+0900    DEBUG   dagstore        dagstore@v0.7.0/dagstore.go:576 restored shard state on dagstore startup        {"shard": "baga6ea4seaqjeouiww2ezctpjh4iqe3huse2qs6sldkstuohxtrhvhsxsae6eaq", "shard state": "ShardStateNew", "shard error": null, "shard lazy": true}
2023-04-24T18:08:01.616+0900    INFO    modules modules/storageminer_idxprov.go:124     Started index provider engine
2023-04-24T18:08:01.618+0900    INFO    dagstore.migrator       dagstore/wrapper.go:301 no shard migration necessary; already marked complete
2023-04-24T18:08:01.704+0900    INFO    modules modules/retrieval.go:193        starting libp2p protocol proxy
2023-04-24T18:08:01.704+0900    INFO    modules modules/storageminer.go:401     waiting for legacy storage provider 'ready' event
2023-04-24T18:08:01.704+0900    INFO    unsmgr  indexprovider/unsealedstatemanager.go:52        starting unsealed state manager running on interval 0s
2023-04-24T18:08:01.704+0900    INFO    modules modules/storageminer.go:410     legacy storage provider started successfully
2023-04-24T18:08:01.704+0900    INFO    modules modules/storageminer.go:413     starting boost storage provider
2023-04-24T18:08:01.704+0900    INFO    index-provider-wrapper  indexprovider/wrapper.go:146    bitswap is not enabled - announcing bitswap disabled to Indexer
2023-04-24T18:08:01.704+0900    INFO    boost-provider  storagemarket/provider.go:389   storage provider: starting
2023-04-24T18:08:01.704+0900    INFO    boost-provider  storagemarket/provider.go:392   db: creating tables
panic: non-positive interval for NewTicker

goroutine 64911 [running]:
time.NewTicker(0x0?)
        /usr/local/go/src/time/tick.go:24 +0x10f
github.com/filecoin-project/boost/indexprovider.(*UnsealedStateManager).Run(0xc0003f8730, {0x67f75e0, 0xc000eff140})
        /boost/indexprovider/unsealedstatemanager.go:53 +0xdd
created by github.com/filecoin-project/boost/indexprovider.NewWrapper.func1.1
        /boost/indexprovider/wrapper.go:86 +0x205
```

### Repo Steps

1. Migrate a monolithic `lotus-miner` to `boost`
2. Try to start `boost`
3. See how it won't be able to start.

**Comment by wisarmy** [comment]
### version
```
boostd version 1.7.5+git.52f2aa6
```
### Steps
1. Migrate a monolithic lotus-miner to boost
2. 
```
boostd --vv migrate-monolith \
       --import-miner-repo=<lotus-miner repo path> \
       --api-sealer=$APISEALER \
       --api-sector-index=$APISECTORINDEX \
       --wallet-publish-storage-deals=$PUBLISH_STORAGE_DEALS_WALLET \
       --wallet-deal-collateral=$COLLAT_WALLET \
       --max-staging-deals-bytes=50000000000 
```
3. boostd --vv run
```
me: 172.16.20.68]: failed to acquire sector {866680 1729} from remote(8): sector not found"}
2023-12-02T11:06:00.482+0800    INFO    dagstore        dagstore@v0.7.0/dagstore.go:281 start: failed shard will recover on next acquire     {"shard": "baga6ea4seaqddxvh4mlg6dtiphwtsfzgrqudesmr3tcriyiip3tqews5l2i7uji", "error": "failed to acquire reader of mount on initialization: failed to unseal deal 44513146: failed to unseal piece from sector 1737: unsealing piece: cannot unseal piece (sector: {{866680 1737} 8}, offset: 0 size: 8522825728) - unsealed cid is undefined"}
2023-12-02T11:06:00.483+0800    INFO    dagstore        dagstore@v0.7.0/dagstore.go:281 start: failed shard will recover on next acquire     {"shard": "baga6ea4seaqd37mzdii6inwjt42rkklyc5k4vcpk5mffqrjjg7zuxtoiybqbeiy", "error": "failed to read/generate CAR Index: The specified key does not exist."}
2023-12-02T11:06:00.483+0800    INFO    dagstore        dagstore@v0.7.0/dagstore.go:281 start: failed shard will recover on next acquire     {"shard": "baga6ea4seaqeyumpvexnwqe7jcsb3s4ubspgtmpjzbmwfsn5e32fxcwrgqzgoly", "error": "failed to read/generate CAR Index: The specified key does not exist."}
2023-12-02T11:06:00.483+0800    INFO    modules modules/storageminer_idxprov.go:149     Started index provider engine
2023-12-02T11:06:00.492+0800    INFO    modules modules/retrieval.go:193        starting libp2p protocol proxy
2023-12-02T11:06:00.492+0800    INFO    modules modules/storageminer.go:370     waiting for legacy storage provider 'ready' event
2023-12-02T11:06:00.492+0800    ERROR   retrieval       impl/provider.go:236    Migrating retrieval provider state machines: never reached target database version
2023-12-02T11:06:00.492+0800    ERROR   markets loggers/loggers.go:63   module initialization error     {"module": "retrieval provider", "err": "never reached target database version"}
2023-12-02T11:06:01.307+0800    INFO    dagstore.migrator       dagstore/wrapper.go:301 no shard migration necessary; already marked complete
2023-12-02T11:06:02.151+0800    INFO    modules modules/storageminer.go:379     legacy storage provider started successfully
2023-12-02T11:06:02.152+0800    INFO    modules modules/storageminer.go:382     starting boost storage provider
2023-12-02T11:06:02.152+0800    INFO    boost-provider  storagemarket/provider.go:390   storage provider: starting
2023-12-02T11:06:02.152+0800    INFO    boost-provider  storagemarket/provider.go:393   db: creating tables
2023-12-02T11:06:02.153+0800    INFO    boost-provider  storagemarket/provider.go:399   db: performing migrations
2023/12/02 11:06:02 goose: no migrations to run. current version: 20230426120000
2023-12-02T11:06:02.154+0800    INFO    boost-provider  storagemarket/provider.go:411   db: initialized
2023-12-02T11:06:02.154+0800    INFO    boost-provider  storagemarket/provider.go:506   storage provider: started
2023-12-02T11:06:02.154+0800    INFO    boost-provider  storagemarket/provider_loop.go:352      provider run loop: start
2023-12-02T11:06:02.154+0800    INFO    modules modules/storageminer.go:388     boost storage provider started successfully
2023-12-02T11:06:02.154+0800    INFO    modules modules/storageminer.go:393     starting boost index provider wrapper
2023-12-02T11:06:02.154+0800    INFO    index-provider-wrapper.boost-migrator   indexprovider/wrapper.go:482    dagstore root is /fastdata/boost/.boost/dagstore
2023-12-02T11:06:02.154+0800    INFO    index-provider-wrapper.boost-migrator   indexprovider/wrapper.go:491    no boost shard migration necessary; already marked complete
2023-12-02T11:06:02.154+0800    INFO    modules modules/storageminer.go:395     boost index provider wrapper started successfully
2023-12-02T11:06:02.154+0800    INFO    unsmgr  indexprovider/unsealedstatemanager.go:52        starting unsealed state manager running on interval 0s
panic: non-positive interval for NewTicker

goroutine 30331 [running]:
time.NewTicker(0x0?)
        /usr/local/go/src/time/tick.go:24 +0x10f
github.com/filecoin-project/boost/indexprovider.(*UnsealedStateManager).Run(0xc00099f180, {0x687bf20, 0xc00099f6d0})
        /root/workspace/boost/indexprovider/unsealedstatemanager.go:53 +0xdd
created by github.com/filecoin-project/boost/indexprovider.(*Wrapper).Start
        /root/workspace/boost/indexprovider/wrapper.go:325 +0x1d9
```
https://github.com/filecoin-project/boost/pull/1596 Is it fixed in version 1.7.5? If not, which version should I use?

**Comment by wisarmy** [comment]
solved  in https://github.com/filecoin-project/boost/issues/1411#issuecomment-1532538444
